icon_definitions.py

```diff
@@ -0,0 +1,10 @@
+REACTION_ACCEPT='✅'
```

Comment:
static content like variables should be defined and reused to minimise duplication and reduce error-prone behaviour